src/Model/GServer.php

```diff
@@ -560,6 +572,9 @@ private static function detect(string $url, string $network = '', bool $only_nod
 				self::detect($url, $network, $only_nodeinfo);
 			}
 			return false;
+		} elseif (Network::isUrlBlocked($url)) {
```

Comment:
The `detect` function is only called via the `check` function that checks for a block in the very first lines. So it shouldn't happen at all that a blocked server could slip through.

Reply:
Thank you for taking care of this.